### PR TITLE
Fix getCommandValues: TrackChanges, PageSize, PageMargin [MACRO-2502]

### PR DESCRIPTION
**Pull request by synoet**
- adds the missing `.uno:ViewTrackChangesInformation` command Value expected by the monorepo.
- safer string access, previous was causing undefined behavior. 
- removes extra quotes from orientation and page color

### CODE REVIEW

File: libreoffice-core/desktop/source/lib/init.cxx [hunk 1]
```diff
@@ -6962,7 +6962,12 @@ static char* doc_getCommandValues(LibreOfficeKitDocument* pThis, const char* pCo
     {
         return getTrackedChanges(pThis);
     }
-
+    // MACRO: {
+    else if (aCommand == ".uno:ViewTrackChangesInformation")
+    {
+        return getTrackedChanges(pThis);
+    }
+    // MACRO: }
```
Comment (chase) [inline libreoffice-core/desktop/source/lib/init.cxx]: Duplicate of `.uno:AcceptTrackedChanges` above, why is this necessary?

**Review by chase — CHANGES_REQUESTED** [review]

File: libreoffice-core/desktop/source/lib/init.cxx (hunk 1 above)
Comment (synoet) [reply libreoffice-core/desktop/source/lib/init.cxx]: It's not necessary, this is just what we have mapped in the monorepo.
```
  VIEW_TRACK_CHANGES_INFORMATION = '.uno:ViewTrackChangesInformation',
```
I assume its because `AcceptTrackedChanges` isn't a very good description of what it returns but I will remove it.

Comment (chase) [reply libreoffice-core/desktop/source/lib/init.cxx]: > It's not necessary, this is just what we have mapped in the monorepo.
> 
> ```
>   VIEW_TRACK_CHANGES_INFORMATION = '.uno:ViewTrackChangesInformation',
> ```
> 
> I assume its because `AcceptTrackedChanges` isn't a very good description of what it returns but I will remove it.

Considering the .uno:Whatever is arbitrary, just change the string assigned to the enum instead. We've done it for other less useful descriptors. Just make sure it doesn't have other uses where that command is used for dispatchCommand for example.

**Review by chase — APPROVED** [review]
